experiment/workdir.py

```diff
@@ -77,5 +77,9 @@ def build_logs_target(self, generated_target_name: str, iteration: int):
     return os.path.join(self.build_logs,
                         f'{generated_target_name}-F{iteration}.log')
 
+  def err_logs_target(self, generated_target_name: str, iteration: int):
```

Comment:
nit: Return type hinting?
Sorry, I just realized that other functions don't have that either, which may have confused you.

I can fix them in later PRs, could you please add this?

Reply:
Sure, can do the same for the rest of:)

Reply:
Probably in a different PR, just to ensure one task per PR : )